File: jesus_sotomayor_rodriguez/Programas/dart/Julio2024/actividad29.dart
class Telefono{
  String? marca;
  int? ram;
  int? rom;
}

class Pc{
  String? cpu;
  int? ram;
  int? almacenamiento;
}

class Camisa{
  int? talla;
  String? marca;
  String? color;
}

void main(){
  Telefono celular = new Telefono();
  celular.marca = 'Samsung';
  celular.ram = 4;
  celular.rom = 128;

  Pc computador = new Pc();
  computador.cpu = 'Ryzen 5 4600g';
  computador.ram = 16;
  computador.almacenamiento = 500;

  Camisa ropa = new Camisa();
  ropa.talla = 8;
  ropa.marca = 'H&M';
  ropa.color = 'Blanco';
}

File: jesus_sotomayor_rodriguez/Programas/ruby/Julio2024/actividad29.rb

class Telefono
    attr_accessor :marca, :ram, :rom
end

class Pc
    attr_accessor :cpu, :ram, :almacenamiento
end

class Camisa
    attr_accessor :talla, :marca, :color
end

celular = Telefono.new
celular.marca = "Samsung"
celular.ram = 4
celular.rom = 128

computador = Pc.new
computador.cpu = "Ryzen 5 4600g"
computador.ram = 16
computador.almacenamiento = 500

ropa = Camisa.new
ropa.talla = 8
ropa.marca = "H&M"
ropa.color = "Blanco"